src/theme/shared/size.ts

```diff
@@ -0,0 +1,9 @@
+const size = {
```

Comment:
The [default size from Chakra-UI ](https://chakra-ui.com/docs/styled-system/theming/theme#sizes):

```js
const sizes = {
    sizes: {
      ...theme.space,
      max: 'max-content',
      min: 'min-content',
      full: '100%',
      '3xs': '14rem',
      '2xs': '16rem',
      xs: '20rem',
      sm: '24rem',
      md: '28rem',
      lg: '32rem',
      xl: '36rem',
    }
}
```

Do you know why there's huge different between our definition and the original one?

As I can see from Figma token, the size listed is referencing to the height of button.

![image](https://user-images.githubusercontent.com/3179244/170169964-2768a020-db6a-4c32-8783-734739b7d5d2.png)

Don't we just need to use the `fontSizes` combines with `spaces`?

Reply:
Yes, it is. I will create a PR with the typography tokens, then we can check if we need to add any size here. Otherwise, I will close this PR.

Reply:
After checking with the designer we will roll out with the two ways. We will add the height sizes as well.
Reason: if we create the element without setting the height and doing  `font size + line height + padding top + padding bottom` is very uncontrollable. We will face this in deeper when extracting the primary button, then we can check which way is the best.